static/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -140,6 +143,7 @@ class AddDashboardWidgetModal extends React.Component<Props, State> {
         dashboards: [],
         userHasModified: false,
         widgetType: WidgetType.DISCOVER,
+        memberList: MemberListStore.loaded ? MemberListStore.getAll() : undefined,
```

Comment:
What happens if you call `getAll` without it being loaded?

Reply:
It should return an empty array `[]`. However, I'm using `undefined` here since I want to distinguish between a loaded but empty `memberList` and an unloaded `memberList` as a conditional for rendering.

Reply:
Actually, I've removed these changes as well since they're no longer necessary due to using `AssigneeSelector` (`AssigneeSelector` will pull from `MemberListStore` itself).